### ISSUE TITLE
Unsupported opcodes and nodetypes

### ISSUE THREAD

**Issue by leehu** [zrax/pycdc#234]
Hi, got some unsupported opcodes:

Unsupported opcode: CALL_FUNCTION_EX
Unsupported opcode: BUILD_TUPLE_UNPACK_WITH_CALL
Unsupported opcode: WITH_CLEANUP_START
Unsupported Node type: 12

thx

**Comment by ortiz-e** [comment]
To add to this, I'm running into this:

Unsupported opcode: BUILD_TUPLE_UNPACK

This has been able to decompile more than any other python decompiler I've tried, so thanks for the hard work!

**Comment by karjok** [comment]
I also had a similar error last night, when I tried to decompile the pyc file compiled with python 3.10, the message "Unsupported opcode: BLA_BLA_BLA" appears.



I think this is an error because the pyc file that will be decoded is a bytecode file compiled with python 3.9 and above, and the opcodes that are not supported by pycdc are opcodes that just appeared in python version 3.9 and above.

so I tried reading the ASTree.cpp file and studying it, then I realized that the opcode that was printed as an error was not in the "case block" of the ASTree.cpp file.

I tried to change it a little by adding the opcodes printed in the error message into the case block (on line 1166)
```cpp
case Pyc::JUMP_IF_NOT_EXC_MATCH_A:
case Pyc::RERAISE_A:
```
then rebuild it. I don't know what happened but it solved my problem when trying to decompile python 3.9 and above bytecode. everything is done!
